frontend/src/app/modules/fields/edit/field-types/text-edit-field.component.ts

```diff
@@ -30,18 +30,7 @@ import {Component} from "@angular/core";
 import {EditFieldComponent} from "core-app/modules/fields/edit/edit-field.component";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/c707e7bf78dbad7cac984139a18fb9d109e530df/frontend/src/app/modules/fields/edit/field-types/text-edit-field.component.ts#L31"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>